### PR TITLE
Review star upgrade

### PR DESCRIPTION
**Pull request by j0ntsang**
As a note, I know seeing more entries in the manifest is scary given the discussions that have been happening since before the break... but fear not! That sort of stuff can / will be written statically on the beta launch of themes. 🤩

### CODE REVIEW

**Review by derek-findthebrink — APPROVED** [review]
:+1:

File: src/sections/reviews.liquid [hunk 1]
```diff
@@ -20,11 +20,9 @@
             <li class="reviews__list-item" data-review-id="{{ forloop.index }}">
               <header>
                 <span class="review__stars">
-                  {% for index in (1..5) %}
+                  {% for index in (1..review.rating) %}
                     <input type="radio" name"rating" value="{{ index }}"  />
```
Comment (highestblue) [inline src/sections/reviews.liquid]: shouldn't this also have a tabindex of -1 if you don't want it to be tabbable?

**Review by highestblue — APPROVED** [review]
Left just one small comment. Looks good to me 👍

File: src/sections/reviews.liquid (hunk 1 above)
Comment (j0ntsang) [reply src/sections/reviews.liquid]: Niiiiiice catch, updated ;)

**Pull request merged**